presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -4692,6 +4707,31 @@ public void testShowTablesLike()
                 .allMatch(tableName -> ((String) tableName).startsWith("or"));
     }
 
+    @Test
+    public void testShowTablesLikeWithEscape()
+    {
+        assertQueryFails("SHOW TABLES IN a LIKE '%$_%' ESCAPE", "line 1:36: no viable alternative at input '<EOF>'");
+        assertQueryFails("SHOW TABLES LIKE 't$_%' ESCAPE ''", "Escape string must be a single character");
+        assertQueryFails("SHOW TABLES LIKE 't$_%' ESCAPE '$$'", "Escape string must be a single character");
+
+        try {
+            assertUpdate("CREATE TABLE test_escape_1 (a bigint)");
+            assertUpdate("CREATE TABLE test_escape11 (a bigint)");
+        }
+        catch (RuntimeException e) {
+            assertTrue(ImmutableSet.of("This connector does not support creating tables",
+                    "Unsupported statement type CreateTable")
+                    .contains(e.getMessage()));
+            return;
```

Comment:
I am not particularly in favor of this lenient approach, as it may mask other (run-time) problems, not just inability to create table.

I would recommend adding overrides to all relevant connectors, just as you did in cassandra case.

(Or, you could introduce (in a separate, preparatory commit) a `supportsCreateTable` method (see `com.facebook.presto.tests.AbstractTestDistributedQueries#supportsViews`). )